app/components/AsideBar.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+import ClosestCities from './ClosestCities';
+import css from '../styles/aside.scss';
+
+// temporary data filler
+const closestCities = [
+    'City1',
+    'City2',
+    'City3',
+    'City4',
+    'City5',
+];
+
+function AsideBar({ className }) {
+    return (
+        <aside className={className}>
+            <ClosestCities
+              title={(<h3>Closest cities to your current location</h3>)}
+              list={closestCities}
+            />
+            {/*ToDo: create a separete component for this*/}
+            <div className="google-map">
```

Comment:
@AlexeyBarkow create separate component for google-map. It should lies into main container, not in the sidebar.